bot.py

```diff
@@ -335,86 +335,87 @@ async def verify(ctx):
+            ('🇩🇪', "de", "Germany", ["German"]),
+            ('🇬🇪', "ge", "Georgia", ["Georgian"]),
+            ('🇬🇷', "gr", "Greece", ["Greek"]),
+            ('🇭🇺', "hu", "Hungary", ["Hungarian"]),
+            ('🇮🇸', "is", "Iceland", ["Icelandic"]),
+            ('🇮🇪', "ie", "Ireland", ["Irish"]),
+            ('🇮🇹', "it", "Italy", ["Italian"]),
+            ('🇽🇰', "xk", "Kosovo", ["Kosovar"]),
+            ('🇰🇿', "kz", "Kazakhstan", ["Kazakh"]),
+            ('🇱🇻', "lv", "Latvia", ["Latvian"]),
+            ('🇱🇮', "li", "Liechteinstein", ["German"]),
+            ('🇱🇹', "lt", "Lithuania", ["Lithuanian"]),
+            ('🇱🇺', "lu", "Luxembourg", ["Luxembourgish", "French", "German"]),
+            ('🇲🇹', "mt", "Malta", ["Maltese"]),
+            ('🇲🇩', "md", "Moldova", ["Romanian"]),
+            ('🇲🇨', "mc", "Monaco", ["French"]),
+            ('🇲🇪', "me", "Montenegro", ["Montenegrin"]),
+            ('🇳🇱', "nl", "Netherlands", ["Dutch"]),
+            ('🇲🇰', "mk", "North Macedonia", ["Macedonian", "Albanian"]),
+            ('🇳🇴', "no", "Norway", ["Norwegian"]),
+            ('🇵🇱', "pl", "Poland", ["Polish"]),
+            ('🇵🇹', "pt", "Portugal", ["Portuguese"]),
+            ('🇷🇴', "ro", "Romania", ["Romanian"]),
+            ('🇷🇺', "ru", "Russia", ["Russian"]),
+            ('🇸🇲', "sm", "San Marino", ["Italian"]),
+            ('🇷🇸', "rs", "Serbia", ["Serbian"]),
+            ('🇸🇰', "sk", "Slovakia", ["Slovak"]),
+            ('🇸🇮', "si", "Slovenia", ["Slovenian"]),
+            ('🇪🇸', "es", "Spain", ["Spanish"]),
+            ('🇸🇪', "se", "Sweden", ["Swedish"]),
+            ('🇨🇭', "ch", "Switzerland", ["German", "French", "Italian"]),
+            ('🇹🇷', "tr", "Turkey", ["Turkish"]),
+            ('🇬🇧', "gb", "United Kingdom", []),
+            ('🇺🇦', "ua", "Ukraine", ["Ukrainian", "Russian"]),
+            ('🇻🇦', "va", "Vatican", ["Italian"]),
+            (':region_asia:', "asia", "Asia", []),
+            (':region_africa:', "africa", "Africa", []),
+            (':region_northamerica:', "us", "North America", []),
+            (':region_northamerica:', "ca", "North America", ["French"]),
+            (':region_northamerica:', 'mx', "North America", ["Spanish"]),
+            (':region_oceania:', "oceania", "Oceania", []),
+            (':region_southamerica:', "south_america", "South America", [])
         ]
 
-        countries = list(emojis.get(ctx.message.content))
+        msgContent = ctx.message.content
+        countries = list(emojis.get(msgContent))
         reg = regex.compile(r"<(:\w+:)\d+>")
-        countries += reg.findall(ctx.message.content)
+        countries += reg.findall(msgContent)
 
         roles_countries = []
         roles_langs_add = []
-        for (emoji, country_name, languages) in db:
-            if emoji in countries:
+        for (emoji, country_code, country_name, languages) in db:
+            if emoji in countries or f" {country_code} " in msgContent:
```

Comment:
Nah, do it with a split as I suggested please
This won't work for this case:
```
;verify es
```
because ` es ` is not a substring of that string

Reply:
Fixed in the latest commit